api/src/utils/passwordCheck.ts

```diff
@@ -0,0 +1,39 @@
+import zxcvbn from 'zxcvbn';
+
+export const passwordCheck = (password: string) => {
```

Comment:
```suggestion
/**
 * Checks if the given password is strong enough.
 * Returns `true` if the password passes the strength test.
 */
export function passwordStrengthCheck(password: string) {
```

Couple things here:
- A nitpick: please prefer function declarations instead of arrow functions. Specially if the functions are named.
- The JSDoc comment (comment starting with `/**`) adds documentation to the function and the ts language server will provide nice hints to it.
- This function should have the logic self contained, as explained in another comment.

About JSDoc:
Here is the main documentation: https://jsdoc.app/
Here is what typescript supports: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html
And it supports inline markdown in the comments.

Reply:
changed to function declaration and naming to strengthCheck, should return a boolean value